tensorflow/compiler/jit/xla_gpu_device.cc

```diff
@@ -53,12 +53,12 @@ xla::StatusOr<std::set<int>> XlaGpuDeviceFactory::ParseVisibleDeviceList(
     if (!absl::SimpleAtoi(platform_gpu_id_str, &platform_gpu_id)) {
       return errors::InvalidArgument(
           "Could not parse entry in 'visible_device_list': '",
-          platform_gpu_id_str, "'. visible_device_list = ",
-          visible_device_list);
+          platform_gpu_id_str,
+          "'. visible_device_list = ", visible_device_list);
     }
     gpu_ids.insert(platform_gpu_id);
   }
-  return gpu_ids;
+  return absl::optional<std::set<int>>(gpu_ids);
```

Comment:
Similarly, I think you should be able to `return {gpu_ids}` or maybe `return {{gpu_ids}}`.

Reply:
Done